people/peter.json

```diff
@@ -0,0 +1,11 @@
+{
+  "key": "peter",
+  "name": "Peter Nutter",
+  "nick": "Peter",
+  "email": "peter.nutter@mild.blue",
+  "title_cs": "JUNIOR DEVELOPER",
+  "title_en": "JUNIOR DEVELOPER",
+  "short_desc_en": "Peter studies Mathematical Informatics at the Faculty of Nuclear Sciences and Physical Engineering at CTU. At Mild Blue, he works as a junior developer. His academic specialization is mathematical analysis, machine learning and neural networks. His hobbies include music and film.",

```

Comment:
He focuses on .... in his academical career. 

Asi lepsi anglicky, ne?

Reply:
jo to zní líp